src/styles/index.scss

```diff
@@ -1,3 +1,6 @@
 body {
```

Comment:
use styles for the body and other resets in a separate file to keep your index file clean